i18n.js

```diff
@@ -2,7 +2,16 @@ module.exports = {
   locales: ["fi", "sv"],
   defaultLocale: "fi",
   pages: {
-    "*": ["common", "projekti", "footer", "error", "header", "breadcrumbs", "tietoa-palvelusta/palautetta-palvelusta-dialog"],
+    "*": [
+      "common",
+      "projekti",
+      "footer",
+      "error",
+      "header",
+      "breadcrumbs",
+      "tietoa-palvelusta/palautetta-palvelusta-dialog",
+      "main-page-sidebar",
+    ],
```

Comment:
Siirrä tuo main-page-sidebar tonne "/" alle. Ei tarvita näitä käännöksiä kaikilla sivuilla.